.Rbuildignore

```diff
@@ -17,12 +22,13 @@ demo/.*\.css
 
 # ignore python files/folders
 setup.py
-index.py
-config.py
+usage.py
+setup.py
 requirements.txt
 MANIFEST.in
 CHANGELOG.md
-tests/
-.pylintrc
+test/
+# CRAN has weird LICENSE requirements
+LICENSE.txt
```

Comment:
Same here, I remember reviewing this and @shammamah had fixed it accordingly.

Reply:
Fixed in d7bd0d46f052ffe8a50697947ef7bddc7194854f